webapp/channels/src/components/suggestion/search_date_suggestion/search_date_suggestion.tsx

```diff
@@ -20,66 +20,63 @@ type Props = SuggestionProps<never> & {
     preventClose?: () => void;
 }
 
-export default class SearchDateSuggestion extends React.PureComponent<Props> {
-    private loadedLocales: Record<string, Locale> = {};
-
-    state = {
-        datePickerFocused: false,
-    };
-
-    handleDayClick = (day: Date) => {
+const IconLeft = () => {
+    return (
+        <i className='icon icon-chevron-left'/>
+    );
+};
+
+const IconRight = () => {
+    return (
+        <i className='icon icon-chevron-right'/>
+    );
+};
+
+const dayPickerIcons = {IconRight, IconLeft};
```

Comment:
Since this is a constant, let's use UPPERCASE_SNAKE_CASE
```suggestion
const DAY_PICKER_ICONS = {IconRight, IconLeft};
```